docs/source/modin/supported/series_str_supported.rst

```diff
@@ -117,9 +117,6 @@ the method in the left column.
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``slice_replace``           | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``replace``                 | P                               |  ``N`` if `pat` is non-string, `repl` is           |
```

Comment:
This entry was redundant (it appears also on line 101, in the correct alphabetical location).